docs/conf.py

```diff
@@ -78,5 +78,4 @@
 html_static_path = ['_static']
 
 # Other HTML settings
-autodoc_member_order = "bysource"
```

Comment:
Why was this one removed? This defaults to alphabetical order, which i think makes it more confusing in the documentation.

Reply:
There were two of them in the file.

Reply:
Haha, oh. That'll do it.